Lib/diffenator/font.py

```diff
@@ -56,6 +57,18 @@ class DFont(TTFont):
     def __init__(self, path=None, lazy=False, size=1500):
         self.path = path
         self.ttfont = TTFont(self.path)
+
+        has_outlines = self.ttfont.has_key("glyf") or self.ttfont.has_key("CFF ")
+        if not has_outlines:
+            # Create faux empty glyf table with empty glyphs to make
+            # it a valid font, e.g. for old-style CBDT/CBLC fonts
```

Comment:
@RoelN why is this needed? I think CBDT/CBLC work without glyf table. Can't find reference in OT spec about glyf being required.

Reply:
He may have added it since Diffenator assumes all fonts need a glyf/cff table (not too sure this is true but seems to make sense).

I'm going through the PR so should have some suggestions regarding this.

Reply:
This was indeed added to patch a bitmap font without glyf/cff tables, but only during runnning the script. It will not save the font, but will make the original diffenator code not choke on a glyphless font.

Reply:
> Diffenator assumes all fonts need a glyf/cff table (not too sure this is true but seems to make sense)

It is not true; OpenType fonts must have a glyph table, but that can be glyf, cff, cbdt, sbix, svg